src/pages/steps/AdditionalInformationStep.tsx

```diff
@@ -200,7 +200,7 @@ AdditionalInformationStep.defaultProps = {
 
 const additionalInformationStepConfiguration: StepsConfiguration<FormDataUnion> = {
   Component: AdditionalInformationStep,
-  title: (t) => t(`movies.create.step5.title`),
+  title: ({ t }) => t(`movies.create.step5.title`),
```

Comment:
Change the title?
Now it's "Laat je film extra opvallen".
Maybe we could use the selected type in this title? Something like ` Laat je concert extra opvallen`

Reply:
I've sent it to @corneelwille, to maybe pickup in a followup ticket

Reply:
Good idea but we can indeed pick that up later. I already created a ticket: https://jira.uitdatabank.be/browse/III-4931. I think it works for `concert` but not for other eventtypes like `sport en beweging`: 
`Laat je sport en beweging extra opvallen` just sounds weird. But that's probably due to the copy of the eventtype, not the idea to make the title dynamic